pkg/reconciler/common/initcontroller.go

```diff
@@ -112,7 +112,7 @@ func (ctrl Controller) fetchSourceManifests(ctx context.Context, opts PayloadOpt
 	case "chains":
 		var chain v1alpha1.TektonChain
 		return AppendTarget(ctx, ctrl.Manifest, &chain)
-	case "results":
+	case "tekton-results":
```

Comment:
Why do we need to change this ?

Reply:
This is wrong because configmapInfo for results is tekton-results instead of results like for other components.